src/backend/Race.java

```diff
@@ -66,8 +66,14 @@ public void updateCarList() {
     }
 
     public void updateCarPositions() {
-        for (int i = 0; i < cars.size(); i++) {
-            cars.get(i).updatePosition(i + 1);
+        if (!isSafetycarDeployed()) {
```

Comment:
Wär es nid übersichtlicher, wenn me das "if" innerhalb vom "for-loop" mache würd?